src/Philter.Aura/Philter.Aura.Web/Twilio/TwilioClientModule.cs

```diff
@@ -0,0 +1,20 @@
+using Philter.Aura.Web.Twilio;
+
+public static class TwilioClientModule
+{
```

Comment:
Here is an example PR in STC of how to inject your TwilioOptions directly, rather than trying to grab it via the services.configure
https://github.com/IntelliTect-dev/StormingTheCastle/pull/1768/files#diff-06fb3fe0bffdf1a0c024c864f0afa42b84d36673ff32e44b3b78aeac9cfe8ef4

Reply:
Nevermind, I can't read code